ajax/libs/monaco-editor/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "monaco-editor",
+  "version": "0.9.0",
+  "description": "A browser based code editor",
+  "filename": "min/vs/editor/editor.main.js",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/monaco-editor.git"
+  },
+  "homepage": "https://microsoft.github.io/monaco-editor/",
+  "keywords": [
+    "Monaco",
+    "Editor",
+    "vscode",
+    "TypeScript",
+    "JavaScript"
+  ],
+  "npmName": "monaco-editor",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(dev|min|min-maps)/**/!(*.json)"
```

Comment:
Since it's not a dist folder, l think use `+(dev|min|min-maps)/**/*.+(js|css|map|svg)` will be better.